src/index.js

```diff
@@ -1067,6 +1068,12 @@ const initialize = async () => {
 
   function handleNewNetwork(networkId) {
     networkDiv.innerHTML = networkId;
+
+    if (networkId === '1') {
+      warningDiv.classList.remove('warning-invisible');
+    } else {
+      warningDiv.classList.add('warning-invisible');
+    }
```

Comment:
We're trying to avoid using the `networkId` as much as possible. We should move this logic to `handleNewChain` and check against the `chainId` instead, which will always be `0x1` for Mainnet.

Reply:
@rekmarks
Resolved - f7844d1e3c08e5258f86f9436f6f71a36862921c